internal/cli/cli.go

```diff
@@ -100,6 +104,18 @@ func (c *banzaiCli) Client() *pipeline.APIClient {
 	return c.client
 }
 
+func (c *banzaiCli) CloudinfoClient() *cloudinfo.APIClient {
+	c.cloudinfoClientOnce.Do(func() {
+		config := cloudinfo.NewConfiguration()
+		config.BasePath = viper.GetString("cloudinfo.basepath")
+		config.UserAgent = "banzai-cli/1.0.0/go"
```

Comment:
Should we increase the cli version number in user agent?

Reply:
We just have to wait some more iterations and we eventually get to 1.0.0 anyways. :)